pyEpiabm/pyEpiabm/utility/transition_time_matrix.py

```diff
@@ -0,0 +1,188 @@
+        matrix.loc['InfectGP', 'Recovered'] =\
+            InverseCdf(pe.Parameters.instance().mean_gp_to_recov,
+                       pe.Parameters.instance().gp_to_recov_icdf)
+        matrix.loc['InfectGP', 'InfectHosp'] =\
+            InverseCdf(pe.Parameters.instance().mean_gp_to_hosp,
+                       pe.Parameters.instance().gp_to_hosp_icdf)
+        matrix.loc['InfectGP', 'Dead'] =\
+            InverseCdf(pe.Parameters.instance().mean_gp_to_death,
+                       pe.Parameters.instance().gp_to_death_icdf)
+        matrix.loc['InfectHosp', 'Recovered'] =\
+            InverseCdf(pe.Parameters.instance().mean_hosp_to_recov,
+                       pe.Parameters.instance().hosp_to_recov_icdf)
+        matrix.loc['InfectHosp', 'InfectICU'] =\
+            InverseCdf(pe.Parameters.instance().mean_hosp_to_icu,
+                       pe.Parameters.instance().hosp_to_icu_icdf)
+        matrix.loc['InfectHosp', 'Dead'] =\
+            InverseCdf(pe.Parameters.instance().mean_hosp_to_death,
+                       pe.Parameters.instance().hosp_to_death_icdf)
+        matrix.loc['InfectICU', 'InfectICURecov'] =\
+            InverseCdf(pe.Parameters.instance().mean_icu_to_icurecov,
+                       pe.Parameters.instance().icu_to_icurecov_icdf)
+        matrix.loc['InfectICU', 'Dead'] =\
+            InverseCdf(pe.Parameters.instance().mean_icu_to_death,
+                       pe.Parameters.instance().icu_to_death_icdf)
+        matrix.loc['InfectICURecov', 'Recovered'] =\
+            InverseCdf(pe.Parameters.instance().mean_icurecov_to_recov,
+                       pe.Parameters.instance().icurecov_to_recov)
+        return matrix
+
+    def update_transition_time_with_float(
+                        self,
+                        current_infection_status_row: enum,
+                        next_infection_status_column: enum,
+                        matrix: pd.DataFrame,
+                        new_transition_time: float):
+        """Method to manually update a transition time in the
+        transition time matrix.
+
+        Parameters
+        ----------
+        current_infection_status_row : enum
+            Infection status corresponding to
+            the row where the probability will be updated
+        next_infection_status_column : enum
+            Infection status corresponding to
+            the column where the probability will be updated
+        matrix : pd.Dataframe
+            Transition time matrix that will have one of it's
+            entries changed
+        new_transition_time : float
+            Updated transition time value
+
+        """
+        try:
+            if (current_infection_status_row not in InfectionStatus) or \
+                    (next_infection_status_column not in InfectionStatus):
+                raise ValueError('Row and column inputs must be contained in\
+                                the InfectionStatus enum')
```

Comment:
Split strings over multiple lines using string addition (ditto for other value errors here)